### PR TITLE
docs: Add "KubeOps" operator SDK to third-party list

### PR DESCRIPTION
**Pull request by buehler**
Adding dotnet operator SDK/framework to third-party list of operator SDKs.

### CODE REVIEW

**Review by sftim — CHANGES_REQUESTED** [review]
Thanks for suggesting this @buehler 

When we add items to _this_ list, we should keep the existing sorting (alphabetical order). Would you be willing to revise this to get the ordering right?

**Comment by buehler** [comment]
Of course! I did not get the ordering, I thought it was by date added.

**Comment by buehler** [comment]
@sftim done

**Comment by reylejano** [comment]
Thanks for fixing the order
/lgtm

File: content/en/docs/concepts/extend-kubernetes/operator.md [hunk 1]
```diff
@@ -114,6 +114,7 @@ Operator.
 
 * [Charmed Operator Framework](https://juju.is/)
 * [kubebuilder](https://book.kubebuilder.io/)
+* [KubeOps](https://buehler.github.io/dotnet-operator-sdk/) (dotnet operator SDK)
```
Comment (celestehorgan) [inline content/en/docs/concepts/extend-kubernetes/operator.md:117]: Small, non-blocking change: dotnet --> .NET

**Comment by reylejano** [comment]
Follow-up PR https://github.com/kubernetes/website/pull/29637 to address [nit](https://github.com/kubernetes/website/pull/29529#discussion_r701241597) of dotnet --> .NET

**Pull request merged**